helm/jupyterhub-home-nfs/templates/deployment.yaml

```diff
@@ -44,6 +45,8 @@ spec:
         volumeMounts:
         - name: home-directories
           mountPath: /export
+        resources:
+          requests: {{ toJson .Values.quotaEnforcer.resources }}
```

Comment:
The `toJson` should be one level up, for `resources` to match. currently resources map to requests.